server/data/prisonerSearchApiClient.ts

```diff
@@ -28,4 +28,14 @@ export default class PrisonerSearchApiClient extends RestClient {
       { username: user?.username }
     )) as Promise<Prisoner[]>
   }
+
+  async searchPrisonersByPrison(prisonCode: string, user?: User): Promise<PagePrisoner> {
+    return (await this.get(
+      {
+        path: `/prisoner-search/prison/${prisonCode}`,
+        query: { size: 2000 },
+      },
+      { username: user?.username }
+    )) as Promise<PagePrisoner>
```

Comment:
If we have any prisons with > 2000 offender in them we probably won't raise alerts? Should we set a smaller page size and iterate over  many pages as there are, or do you think this will suffice?